certified-connectors/tegolySIGN/readme.md

```diff
@@ -0,0 +1,12 @@
+## tegolySIGN Connector
+Automate your signatures with the tegolySIGN connector. You can transform manual processes into fully digital ones. Integrate tegolySIGN functions into your favourite applications and create fully automated signature workflows.
```

Comment:
@JanGosda all files should free of grammatical errors and typos. please change "favourite" to "favorite"